connectors/sources/sharepoint_online.py

```diff
@@ -406,28 +445,14 @@ async def site_drives(self, site_id):
             for site_drive in page:
                 yield site_drive
 
+    # Use Delta queries
```

Comment:
I would inline this comment to the method making it a docstring instead.

Reply:
Yeah, this one I just forgot to remove. I'll add more comments using docstrings and this one will be just removed.